src/Nest/Modules/SnapshotAndRestore/Snapshot/SnapshotStatus/SnapshotStatusResponse.cs

```diff
@@ -24,8 +24,12 @@ public class SnapshotStatus
 		public string Snapshot { get; internal set; }
 		[JsonProperty("repository")]
 		public string Repository { get; internal set; }
+		[JsonProperty("uuid")]
+		public string UID { get; internal set; }
 		[JsonProperty("state")]
 		public string State { get; internal set; }
+		[JsonProperty("include_global_state")]
+		public bool IncludeGlobalState { get; internal set; }
```

Comment:
Should this be `bool?`, considering Elasticsearch < 6.2.0 do not return this value?

Reply:
👍

Reply:
++